.actions/app/delete_cloud_lightning_apps.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
This is not related to the docs, should be in another PR. I added it with the e2e tests 😄 

https://github.com/Lightning-AI/lightning/pull/13499/files#diff-25803814a480f40540e95e6dc62c8dd0995ad5eedfbf858ab5a28d52c27516b6R331

Reply:
Removed.